tests/smoke_test.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+import freetype
+import pytest
+
+
+def test_load_ft_face():
+    """A smoke test."""
+    assert freetype.Face('../examples/Vera.ttf')
+
+
+def test_bundle_version():
+    if os.environ.get("FREETYPEPY_BUNDLE_FT"):
```

Comment:
hm, that environment variable is only needed at setup time, should not change anything when running tests. 
to detect if running freetype-py with a bundled library you should rather check that there is one at the expected path.

Reply:
What if for whatever reason a `.so` isn't present when it should? Testing for the environment variable makes it clear that there must be something.

Reply:
> What if for whatever reason a .so isn't present when it should?

then something went wrong in the installation and should fail!

> Testing for the environment variable makes it clear that there must be something.

the env variable isn't required at runtime (and hence, at test time), only when you run setup.py.

Reply:
just skip the test if it's not present

Reply:
I mean, is it enough to os.path.exists(glob.glob(packagedir / libfreetype*)) in the test? I suppose setup will fail if the env var is set but no .so is put anywhere, but what if it doesn't under some circumstance?